### DISCUSSION TITLE
Auto-zoom basemap

### DISCUSSION THREAD

**Discussion by hski-github** [neocarto/bertin#74] [Q&A]
If I use a world map as basemap and have sales data for a couple of countries, it is possible that bertin zooms the world map automatically so that the map is zoomed to the countries with data in another layer? Sorry, guess this is a beginners question, but I am a beginner. :-)

**Comment by neocarto** [comment]
Yes, you can define the extent in the parameters.
See the map on africa in this notebook.
https://observablehq.com/@neocartocnrs/hello-bertin-js?collection=@neocartocnrs/bertin
